.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,18 @@
+name: "Static analysis"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+      - name: "PHP-CS-Fixer"
+        uses: "docker://oskarstark/php-cs-fixer-ga:2.19.0"
```

Comment:
looking at https://github.com/php-http/HttplugBundle/pull/399/checks?check_run_id=3461567716 it seems like something is off. it says that it "fixed" a file, but i guess it should error and report what needs to be fixed rather than silently fix during the run, and then lose the change?